firehose.go

```diff
@@ -143,6 +143,6 @@ func (p *Producer) sendFirehoseRecords(args *gokinesis.RequestArgs) {
 	}
 
 	if conf.Debug.Verbose && p.getMsgCount()%100 == 0 {
-		log.Println("Messages sent so far: " + strconv.Itoa(p.getMsgCount()))
+		log.Println("Messages sent so far: " + strconv.FormatInt(p.getMsgCount(), 10))
```

Comment:
`Itoa` only works for `int` types.  To use it for int64 (which is what an atomic counter uses), we must use longhand.